src/pcrlock/pcrlock.c

```diff
@@ -2207,6 +2212,10 @@ static int show_pcr_table(EventLog *el, JsonVariant **ret_variant) {
         (void) table_set_json_field_name(table, 7, "noMissingComponents");
 
         for (uint32_t pcr = 0; pcr < TPM2_PCRS_MAX; pcr++) {
+
+                if (!FLAGS_SET(pcr_mask,  UINT32_C(1) << pcr))
+                        continue;
+
                 /* Check if the PCR hash value matches the event log data */
                 bool hash_match = event_log_pcr_checks_out(el, el->registers + pcr);
```

Comment:
hmm, i think limiting things like this is OK, but I think this should be opt-in, we shouldn't hide stuff that actually might be useful to lock down too.

(another idea: show it, but grey it out?)

Reply:
ah, misleading code. The idea is basically
`if (arg_pcr_mask && !FLAGS_SET(arg_pcr_mask,  UINT32_C(1) << pcr))`
which is what I should put here